src/Plugin.cc

```diff
@@ -141,8 +141,8 @@ FILE* open_map_file() {
 // <prefix><function_name>:<filename>:<first_line>
 std::string format_map_entry(const std::string& prefix, const zeek::Func* f, const zeek::detail::StmtPtr& stmt) {
     const auto* loc = stmt->GetLocationInfo();
-    auto fn = zeek::util::detail::without_zeekpath_component(loc->filename);
-    std::string loc_str = zeek::util::fmt("%s:%d", fn.c_str(), loc->first_line);
+    auto fn = zeek::util::detail::without_zeekpath_component(loc->FileName());
+    std::string loc_str = zeek::util::fmt("%s:%d", fn.c_str(), loc->FirstLine());
```

Comment:
Could you ifdef this with two branches so it works against both the LTS release and Zeek `master`?